src/main/java/gov/nasa/jpl/mbee/mdk/util/Utils.java

```diff
@@ -1756,6 +1759,18 @@ public static void displayValidationWindow(Project project, Collection<Validatio
 
         Package dummyvs = (Package) project.getElementByID("_17_0_2_407019f_1354124289134_280378_12909");
         if (dummyvs == null) {
+            try {
```

Comment:
This is out of scope for this issue and requires extensive testing. Please revert for now.